addons/account/wizard/account_secure_entries_wizard.py

```diff
@@ -0,0 +1,268 @@
+                chain_info['not_hashable_unlocked_moves'] = not_hashable_unlocked_moves
+                res.append(chain_info)
+        return res
+
+    @api.depends('company_id', 'company_id.user_hard_lock_date', 'hash_date')
+    def _compute_data(self):
+        for wizard in self:
+            unreconciled_bank_statement_line_ids = []
+            chains_to_hash = []
+            if wizard.hash_date:
+                for chain_info in wizard._get_chains_to_hash(wizard.company_id, wizard.hash_date):
+                    if 'unreconciled' in chain_info['warnings']:
+                        unreconciled_bank_statement_line_ids.extend(
+                            chain_info['moves'].statement_line_ids.filtered(lambda l: not l.is_reconciled).ids
+                        )
+                    else:
+                        chains_to_hash.append(chain_info)
+            wizard.unreconciled_bank_statement_line_ids = [Command.set(unreconciled_bank_statement_line_ids)]
+            wizard.chains_to_hash_with_gaps = [
+                {
+                    'first_move_id': chain['moves'][0].id,
+                    'last_move_id': chain['moves'][-1].id,
+                } for chain in chains_to_hash if 'gap' in chain['warnings']
+            ]
+
+            not_hashable_unlocked_moves = []
+            move_to_hash_ids = []
+            for chain in chains_to_hash:
+                not_hashable_unlocked_moves.extend(chain['not_hashable_unlocked_moves'].ids)
+                move_to_hash_ids.extend(chain['moves'].ids)
+            wizard.not_hashable_unlocked_move_ids = [Command.set(not_hashable_unlocked_moves)]
+            wizard.move_to_hash_ids = [Command.set(move_to_hash_ids)]
+
+    @api.depends('company_id', 'chains_to_hash_with_gaps', 'hash_date', 'not_hashable_unlocked_move_ids', 'max_hash_date', 'unreconciled_bank_statement_line_ids')
+    def _compute_warnings(self):
+        for wizard in self:
+            warnings = {}
+
+            if wizard.max_hash_date:
+                max_hash_date_string = format_date(self.env, wizard.max_hash_date)
+            else:
+                max_hash_date_string = _("today")
+            warnings['account_max_hash_date'] = {
+                'message': _("Posted entries are currently secured up to %s, inclusive.", max_hash_date_string),
+            }
+
+            if not wizard.hash_date:
+                wizard.warnings = warnings
+                continue
+
+            if wizard.unreconciled_bank_statement_line_ids:
+                ignored_sequence_prefixes = list(set(wizard.unreconciled_bank_statement_line_ids.move_id.mapped('sequence_prefix')))
+                warnings['account_unreconciled_bank_statement_line_ids'] = {
+                    'message': _("There are still unreconciled bank statement lines before the selected date. "
+                                 "The entries from journal prefixes containing them will not be secured: %(prefix_info)s",
```

Comment:
We raise an error instead of ignoring, right?
```suggestion
                                 "The entries from journal prefixes containing them can not be secured: %(prefix_info)s",
```